plugins/by-name/diffview/default.nix

```
@@ -1,145 +1,138 @@
 {
   lib,
-  helpers,
   config,
-  pkgs,
-  options,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.diffview;
-  mkWinConfig =
-    {
-      type ? null,
-      width ? null,
-      height ? null,
-      position ? null,
-    }:
-    with helpers.defaultNullOpts;
-    {
-      type =
-        mkEnum
-          [
-            "split"
-            "float"
-          ]
-          type
-          ''
-            Determines whether the window should be a float or a normal
-            split.
-          '';
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
```

This was renamed to:

```suggestion
lib.nixvim.plugins.mkNeovimPlugin {
```